django/utils/crypto.py

```diff
@@ -17,7 +18,11 @@
 
 # Use the system PRNG if possible
 try:
-    random = random.SystemRandom()
+    if sys.version_info >= (3, 6):
+        import secrets
+        random = secrets.SystemRandom()
+    else:
+        random = random.SystemRandom()
```

Comment:
Presumably the intention is that `secrets` might one day use a different PRNG's on some OS's